tests/general/copyrightcheck_test_files/copyright_without_author.java

```diff
@@ -0,0 +1,10 @@
+//Copyright (C) 2014 - A copyright notice
+
+public class copyright_without_author {
```

Comment:
@nvzard  I am accepting the point that `travis ` build error log is because of `DDOS` problem but the `CIRCLE CI` log on current PR is because of cache problem ,not 100% but 90% sure It happened with me many time, to solve it there is clear message in documentation to disable `Circle CI` on fork,

@nvzard  `Compare error log of Circle and Travis CI`  both are different.

Reply:
@RaiVaibhav `Circle CI` is also showing sourceforge issue now.
@nvzard fixed the class name.

Reply:
Nice :+1: